templates/layouts/identified.jade

```diff
@@ -103,11 +106,12 @@ block content
                         a(href='/search/modifiedBefore='+meta.modified.description.split(' ')[0]+'&modifiedAfter='+meta.modified.description.split(' ')[0]+'&',title='Find all records modified on this date') 
                             span.fa.fa-search
 
-                if meta.description && meta.description != ''
+                if (meta.description && meta.description != '') || meta.canUpdate 
                     h4(class=meta.canUpdate ? "edit edit-description" : "")
-                        != meta.description 
-                        a(href='/search/?q='+meta.description,title='Find all records with terms in common with this description')
-                            span.fa.fa-search
+                        != meta.description
+                        if meta.description && meta.description != ''
+                            a(href='/search/?q='+meta.description.replace('<br/>',' '),title='Find all records with terms in common with this description')
```

Comment:
Is this supposed to be part of this PR?

Reply:
Yeah, this was the problem with extra breaks causing search by description to be broken.  I guess I could have put in separate PR, but I discovered as I was fixing these other issues.